install-pcf/gcp/pipeline.yml

```diff
@@ -276,6 +276,7 @@ jobs:
       db_silk_password: {{db_silk_password}}
       db_locket_username: {{db_locket_username}}
       db_locket_password: {{db_locket_password}}
+      BLOBSTORE_INTERNAL_ACCESS_RULES: {{blobstore_internal_access_rules}}
```

Comment:
I have remove BLOBSTORE_INTERNAL_ACCESS_RULES in GCP. This is not implemented at all in AWS.